src/pybind/mgr/dashboard/frontend/src/app/shared/components/text-label-list/text-label-list.component.ts

```diff
@@ -0,0 +1,101 @@
+import { Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR, ReactiveFormsModule } from '@angular/forms';
+import { ButtonModule, GridModule, IconModule, InputModule } from 'carbon-components-angular';
+import { ComponentsModule } from '../components.module';
+
+@Component({
+  selector: 'cd-text-label-list',
+  standalone: true,
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+    InputModule,
+    IconModule,
+    ButtonModule,
+    GridModule,
+    ComponentsModule
+  ],
+  templateUrl: './text-label-list.component.html',
+  styleUrl: './text-label-list.component.scss',
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: TextLabelListComponent,
+      multi: true
+    }
+  ]
+})
+export class TextLabelListComponent implements ControlValueAccessor {
+  @Input()
+  label: string = '';
+  @Input()
+  helperText: string = '';
+  @Input()
+  required: boolean = false;
```

Comment:
```suggestion
  isRequired: boolean = false;
```

Reply:
removed the input since was only useful to toggle helperText/required text but the toggle it's currently not working